src/hddsupport.c

```diff
@@ -395,13 +404,16 @@ static void hddLaunchGame(int id, config_set_t *configSet)
                 }
 
                 if (have_error) {
-                    char error[256];
-                    if (have_error == 2) // VMC file is fragmented
-                        snprintf(error, sizeof(error), _l(_STR_ERR_VMC_FRAGMENTED_CONTINUE), vmc_name[vmc_id], (vmc_id + 1));
-                    else
-                        snprintf(error, sizeof(error), _l(_STR_ERR_VMC_CONTINUE), vmc_name[vmc_id], (vmc_id + 1));
-                    if (!guiMsgBox(error, 1, NULL))
-                        return;
+                    if (gAutoLaunchGame == NULL) {
+                        char error[256];
+                        if (have_error == 2) // VMC file is fragmented
+                            snprintf(error, sizeof(error), _l(_STR_ERR_VMC_FRAGMENTED_CONTINUE), vmc_name[vmc_id], (vmc_id + 1));
+                        else
+                            snprintf(error, sizeof(error), _l(_STR_ERR_VMC_CONTINUE), vmc_name[vmc_id], (vmc_id + 1));
+                        if (!guiMsgBox(error, 1, NULL))
+                            return;
+                    } else
+                        LOG("VMC error\n");
```

Comment:
Should there be a return here for error handling?

Reply:
Nah it should be fine, the return there is caused by user input deciding they don’t want to continue because their vmc failed. In the case of auto booting we don’t have much choice but to continue without the vmc since there is no GUI to go back to.

Reply:
Ah, I see